docs/docsite/rst/reference_appendices/glossary.rst

```diff
@@ -250,6 +250,11 @@ when a term comes up on the mailing list.
         Ansible uses JSON for return data from remote modules.  This allows
         modules to be written in any language, not just Python.
 
+    Keyword
+        The main expressions that make up Ansible, which apply to playbook objects
+        (Play, Block, Role and Task). For example 'vars:' is a keyword that lets
+        you define variables in the scope of the playbook object it is applied to.
```

Comment:
Thanks for clarifying that, that makes more sense to me in the use of `role` keyword for instance.